primitives/src/locktime/relative.rs

```diff
@@ -227,6 +260,18 @@ impl LockTime {
     /// When deciding whether an instance of `<n> CHECKSEQUENCEVERIFY` will pass, this
     /// method can be used by parsing `n` as a [`LockTime`] and calling this method
     /// with the sequence number of the input which spends the script.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use bitcoin_primitives::{Sequence, relative::LockTime};
+    ///
+    /// let sequence = Sequence::from_consensus(7000);
+    /// let lock_time = LockTime::from_sequence(sequence).unwrap();
+    /// let input_sequence = Sequence::from_consensus(8000);
```

Comment:
```suggestion
    /// let sequence = Sequence::from_consensus(7_000);
    /// let lock_time = LockTime::from_sequence(sequence).unwrap();
    /// let input_sequence = Sequence::from_consensus(8_000);
```

Reply:
I changed these to time values to show a different usage to the previous example that used a block height.